### ISSUE TITLE
Allow physical metadata to be defined in a template

### ISSUE THREAD

**Issue by clnsmth** [EDIorg/EMLassemblyline#107]
`make_eml()` requires access to data files so physical metadata (e.g. file size, checksum, etc.) can be calculated and the user doesn't have to deal with this minutiae. In some cases this requirement is unnecessarily prohibitive (e.g. large unchanged data files stored in a remote location) or prevents the user from overriding EAL assumptions (e.g. formatName estimated from file extension).

A possible solution is to store this physical metadata in a new metadata template that can be created before running `make_eml()` or left absent in the case where a user wants the values calculated for them, as is currently done.

**Comment by clnsmth** [comment]
The proposed template is https://docs.google.com/spreadsheets/d/1dMbGtbmfVzUWUR0TyMUFHtkdrIckXc_-nRwKpoqkUmY/edit?usp=sharing

**Comment by joefutrelle** [comment]
Looks good to me. In the scenario where we transfer files to you out of band (because we can't host them), would the other entity URL be left blank? Or does EML require it?

**Comment by clnsmth** [comment]
You're right @joefutrelle, in that case the URL field would be blank or `NA` so the EDI Repository wouldn't try downloading the associated data object.

**Comment by joefutrelle** [comment]
Perfect.
